libflux/core/src/lib.rs

```diff
@@ -18,11 +19,15 @@ pub mod semantic;
 
 use std::error;
 pub use std::fmt;
+use std::hash::BuildHasherDefault;
 
 use derive_more::Display;
+use fnv::FnvHasher;
 
 pub use ast::DEFAULT_PACKAGE_NAME;
 
+type FluxHasher = BuildHasherDefault<FnvHasher>;
```

Comment:
The name FluxHasher made me think this was a hash algo specific to Flux in some way. Maybe call it DefaultHasher? It's in the flux/core crate so it already has the context of it being specific to the Flux code base. Thoughts?

Reply:
Yeah, I think that's a good idea. I think `DefaultHasher` is a good name.